### ISSUE TITLE
homepage design improvements

### ISSUE THREAD

**Issue by 7yl4r** [marinebon/www_marinebon2#32]
* [x] borrow homepage design from https://ioos.noaa.gov/
    * [ ] overview header that scrolls away then top nav (that is going to be hard to do)
    * [x] lighter blue section under the nav with desc

**Comment by 7yl4r** [comment]
* [ ] add white background image above the nav
* [ ] simplify the MBON logo in top nav
* [ ] try to find and copy over the javascript the "ark-topbar"

This above-top-nav addition will mean that we don't need to worry about making the top nav taller.

**Comment by 7yl4r** [comment]
The javascript that controls the hiding above-navbar seems complicated.
I found a file named `auto-hiding-navbar.js` which sounds promising but it operates on elements with the class `auto-hiding-navbar` only and that class is not in the page html.

If this is handled by the ark framework then it would be in [this big js file](https://ioos.noaa.gov/wp-content/themes/ark/assets/scripts/app.js?ver=f35c911899b287d4e1e4fd5ba667df2e). 

What I see happening in the html inspector is a small change in the div:

```html
<!-- before scrolling -->
<header class="ark-header ark-header-mobile-fixed header-no-pills header header-sticky navbar-fixed-top header-has-topbar fg-text-dark ffb-header-design">

<!-- after scrolling -->
<header class="ark-header ark-header-mobile-fixed header-no-pills header header-sticky navbar-fixed-top header-has-topbar fg-text-dark ffb-header-design header-shrink">
```

so basically `header-shrink` gets added to the list of classes and this triggers a CSS transition that collapses the bar.
That css appears to live in the ark `style.css` (big file) [here](https://ioos.noaa.gov/wp-content/themes/ark/style.css?ver=f35c911899b287d4e1e4fd5ba667df2e).

**Comment by torieketcham** [comment]
I've created a logo file for the white background area above the nav - it's attached. I can work on the smaller logo within the nav as well. I'll probably simplify it so that it just has the MBON lettering and nothing else.

 Is there anything you wanted me to do re: accessing the ark files?

![mbon-logo](https://user-images.githubusercontent.com/44508950/132051406-30ce1543-6f5c-4186-8585-2f29c0fa5cc3.png)

**Comment by 7yl4r** [comment]
I think I have all the ark files I would need but it is a bit of work to get this done. The existing nav at the top is positioned dynamically too - so I think we need to rewrite the js+css that does that too.
